### ISSUE TITLE
./configure --prefix support

### ISSUE THREAD

**Issue by calidion** [SoftEtherVPN/SoftEtherVPN#160]
currently --prefix seems not supported.

**Comment by chipitsine** [comment]
build system has evolved a lot. it uses CMake now. it is possible to implement --prefix feature, do we need to write document on that, @davidebeatrici  ?

**Comment by davidebeatrici** [comment]
Yes, we should document all the options we implement for `configure`.
